src/Plugin/views/field/CollaboraEdit.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\collabora_online\Plugin\views\field;
+
+use Drupal\collabora_online\Cool\CoolUtils;
+use Drupal\Core\Url;
+use Drupal\views\Plugin\views\field\LinkBase;
+use Drupal\views\ResultRow;
+
+/**
+ * Field handler for link to edit a collabora file.
+ *
+ * @ingroup views_field_handlers
+ *
+ * @ViewsField("media_collabora_edit")
+ */
```

Comment:
We can use attributes.
They are available since 10.3.0 which matches our core requirement in composer.json.
```suggestion
 */
#[ViewsField('media_collabora_edit')]
```

Reply:
True